client/src/component_home/BookList/BookList.jsx

```diff
@@ -0,0 +1,22 @@
+import PropTypes from "prop-types";
+import BookCard from "./BookCard";
+import styles from "./BookList.module.css";
+
+function BookList({ bookList }) {
+  return (
+    <div className={styles.homeBooks}>
+      {Array.isArray(bookList) &&
```

Comment:
soyez explicites dans vos conditions, par exemple :  
Array.isArray{bookList} === true

Pensez bien que lorsque vous utilsez des fonctions qui commencent par 'is', par convention soit elles renvoient false soit true